qemu/tests/qmp_basic.py

```diff
@@ -332,7 +335,10 @@ def argument_checker_suite(monitor):
 
         # value argument must be a json-number
         for arg in ({}, [], True, "foo"):
-            resp = monitor.cmd_qmp("migrate_set_speed", {"value": arg})
+            if utils_misc.compare_qemu_version(5, 1, 0) is True:
+                resp = monitor.cmd_qmp("migrate-set-parameters", {"value": arg})
```

Comment:
To replace `migrate-set-speed` with `migrate-set-parameters`, parameter name `max-bandwidth` is must

Reply:
resp = monitor.cmd_qmp("migrate-set-parameters", {"max-bandwidth": arg})